src/Layer/Layer.js

```diff
@@ -22,6 +22,14 @@ import Cache from 'Core/Scheduler/Cache';
  * @property {boolean} [labelEnabled=false] - Used to tell if this layer has
  * labels to display from its data. For example, it needs to be set to `true`
  * for a layer with vector tiles.
+ * @property {object} [zoom] - This property is used only the layer is attached to [TiledGeometryLayer]{@link TiledGeometryLayer}.
+ * The layer checks the tile zoom level to determine if the layer is visible in this tile.
+ *
+ * ![tiled geometry](/docs/static/images/wfszoommaxmin.jpeg)
```

Comment:
Great addition ! Can you add a legend/description to tell that the tiles in question are red or orange, and the different levels between the top ones and the bottom one ? Then it is good with me, but please wait for an opinion of @jailln before merging